README.md

```diff
@@ -125,7 +127,7 @@ To understand when the lifecycle methods are called take a look at the following
 $ gem install cocoapods
 ```
 
-> CocoaPods 0.39.0+ is required to build BothamUI 1.0.0+.
+> CocoaPods 0.39.0+ is required to build BothamUI.
```

Comment:
I've removed this reference to the library version to don't forget to update it when the version number changes xD